src/constants.js

```diff
@@ -5,5 +5,13 @@ module.exports = {
   NETWORK: 'bitcoin',
   getNetwork: function () {
     return Bitcoin.networks[this.NETWORK];
+  },
+  getDefaultWalletOptions: function () {
```

Comment:
Yes, as a function it returns a new object every time. If it were a direct reference to an object, it would be exposed for mutations, which I'd like to guard against.